src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -164,6 +164,7 @@ def handle_message(
                     # XXX(ahmed): Remove this logic. This was kept here as backwards compat
                     # for subscription updates with schema version `2`. However schema version 3
                     # sends the "entity" in the payload
+                    logger.info("Message value using version 2")
```

Comment:
I'd make this warn or error level. Shouldn't ever really happen. I think the Sentry logging integration also means that it gets forwarded to Sentry that way.